Frequently Asked Questions and Guidelines.md

```diff
@@ -0,0 +1,47 @@
+# Frequently Asked Questions and Guidelines
```

Comment:
We have `faq.md` are you aware of that file? If not, do you think this document would more fit there?

Reply:
Hi, I haven't seen that faq.md document you are right that topic need to be in that document. I will append that document. Also as I didn't change anything on the webpage I don't need to get any preview, do I ?

Reply:
Hi @volkansahn, ah I think I misunderstood you.

Actually, we have a dedicated FAQ which available to view mdwiki (web page) as well, so a preview URL still needed.